paymentsheet/src/test/java/com/stripe/android/paymentsheet/paymentdatacollection/cvcrecollection/DefaultCvcRecollectionInteractorTest.kt

```diff
@@ -0,0 +1,49 @@
+package com.stripe.android.paymentsheet.paymentdatacollection.cvcrecollection
+
+import app.cash.turbine.test
+import com.google.common.truth.Truth.assertThat
+import com.stripe.android.model.CardBrand
+import kotlinx.coroutines.test.runTest
+import org.junit.Test
+
+class DefaultCvcRecollectionInteractorTest {
+    private fun createInteractor(): DefaultCvcRecollectionInteractor {
+        return DefaultCvcRecollectionInteractor(
+            args = Args(
+                lastFour = "4242",
+                cardBrand = CardBrand.Visa,
+                cvc = null,
+                isTestMode = true
+            ),
+        )
+    }
+
+    @Test
+    fun `view model state initialized properly on init`() {
+        val interactor = createInteractor()
+
+        assertThat(interactor.viewState.value).isEqualTo(
+            CvcRecollectionViewState(
+                cardBrand = CardBrand.Visa,
+                lastFour = "4242",
+                cvc = null,
+                isTestMode = true
+            )
+        )
+    }
```

Comment:
can you add a test that the state gets updated properly when there is a cvc state view action?